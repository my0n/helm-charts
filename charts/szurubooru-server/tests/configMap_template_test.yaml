suite: test configMap
templates:
  - configMap.yaml
tests:
  - it: should match snapshot
    chart:
      version: 1.0.0
      appVersion: 1.0.0
    set:
      configuration.enabled: true
      configuration.template: |
        name: My Self-Hosted Szurubooru
        secret: ${SZURUBOORU_SECRET}
      configuration.templateEnv:
        - name: SZURUBOORU_SECRET
          valueFrom:
            secretKeyRef:
              name: szurubooru-secret
              key: secret
    asserts:
      - matchSnapshot: {}