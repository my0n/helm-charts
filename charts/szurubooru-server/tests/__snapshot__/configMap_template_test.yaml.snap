should match snapshot:
  1: |
    apiVersion: v1
    data:
      config.yaml: |
        # configuration file generated by helm
        name: My Self-Hosted Szurubooru
        secret: ${SZURUBOORU_SECRET}
    kind: ConfigMap
    metadata:
      name: RELEASE-NAME-szurubooru-server
